tests/rt.sh

```diff
@@ -304,30 +304,29 @@ elif [[ $MACHINE_ID = acorn ]]; then
   PTMP=/lfs/h2/emc/ptmp
   SCHEDULER=pbs
 
-elif [[ $MACHINE_ID = gaea-c5 ]]; then
+elif [[ $MACHINE_ID = gaea ]]; then
 
-  module use /lustre/f2/dev/role.epic/contrib/C5/rocoto/modulefiles
+  module use /ncrc/proj/epic/rocoto/modulefiles
   module load rocoto
   ROCOTORUN=$(which rocotorun)
   ROCOTOSTAT=$(which rocotostat)
   ROCOTOCOMPLETE=$(which rocotocomplete)
   ROCOTO_SCHEDULER=slurm
 
+  
   module load PrgEnv-intel/8.3.3
   module load intel-classic/2023.1.0
-  module load cray-mpich/8.1.25
```

Comment:
can you please add `module load cray-mpich/8.1.25` back?

Reply:
@ulmononian ok module added back.